.rubocop.yml

```diff
@@ -11,7 +11,7 @@ require:
 
 AllCops:
   DisabledByDefault: true
-  TargetRubyVersion: 2.6
+  TargetRubyVersion: 2.7
```

Comment:
```suggestion
  TargetRubyVersion: 3.0
```

Reply:
3.0 may suggest changes that are not compatible with 2.7, actually breaking 2.7 compatibility

Reply:
Thanks! Then we can leave the Rubocop config as it was. I don't care about having Ruby 2.7 in CI since its EOL. It's best-case scenario for others still on it that want to use it whether it works for them. It's intentional to not run Ruby 2.7 in CI.

Reply:
Done and clarified in the commit message why the CI will not run against 2.7

Reply:
Thanks!